### PR TITLE
UHM-4226 Visit Note smoke tests intermittently failing

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-4226

Fix by using clickOn in more places.

### CODE REVIEW

File: src/main/java/org/openmrs/module/mirebalais/smoke/pageobjects/AppDashboard.java [hunk 1]
```diff
@@ -16,11 +16,7 @@
 
 import org.openmrs.module.mirebalais.apploader.CustomAppLoaderConstants;
 import org.openmrs.module.mirebalais.smoke.dataModel.Patient;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.StaleElementReferenceException;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```
Comment (mogoodrich) [inline src/main/java/org/openmrs/module/mirebalais/smoke/pageobjects/AppDashboard.java:19]: Global import?

File: src/test/java/org/openmrs/module/mirebalais/smoke/BasicMirebalaisSmokeTest.java [hunk 2]
```diff
@@ -18,10 +18,7 @@
 import org.openmrs.module.mirebalais.smoke.pageobjects.LoginPage;
 import org.openmrs.module.mirebalais.smoke.pageobjects.MirebalaisLoginPage;
 import org.openmrs.module.mirebalais.smoke.pageobjects.VisitNote;
-import org.openqa.selenium.OutputType;
-import org.openqa.selenium.TakesScreenshot;
-import org.openqa.selenium.TimeoutException;
-import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.*;
```
Comment (mogoodrich) [inline src/test/java/org/openmrs/module/mirebalais/smoke/BasicMirebalaisSmokeTest.java:21]: Another global?

**Comment by brandones** [comment]
@mogoodrich Yeah I guess my IntelliJ Code Style config was off. I assume we use the OpenMRS Maven Formatter style XML? Though maybe we aren't using the maven formatter itself, at least not consistently? Is there a way to load that XML into IntelliJ?

**Comment by mogoodrich** [comment]
Not sure re: loading that XML into IntelliJ... I believe I just changed that one styling parameter.

**Pull request merged**